libs/dh/esett/feature-outgoing-messages/src/lib/filters/dh-filters.component.ts

```diff
@@ -114,17 +119,32 @@ export class DhOutgoingMessagesFiltersComponent implements OnInit {
   formGroup!: FormGroup<Filters>;
 
   ngOnInit(): void {
+    const {
+      calculationTypes,
+      actorNumber,
+      created,
+      gridAreas,
+      latestDispatch,
+      messageTypes,
+      period,
+      status,
+    } = this.initial();
+
     this.formGroup = new FormGroup<Filters>({
-      calculationTypes: dhMakeFormControl(this.initial?.calculationTypes),
-      messageTypes: dhMakeFormControl(this.initial?.messageTypes),
-      gridAreas: dhMakeFormControl(this.initial?.gridAreas),
-      actorNumber: dhMakeFormControl(this.initial?.actorNumber),
-      status: dhMakeFormControl(this.initial?.status),
-      period: dhMakeFormControl(this.initial?.period),
-      created: dhMakeFormControl(this.initial?.created),
-      latestDispatch: dhMakeFormControl(this.initial?.latestDispatch),
+      calculationTypes: dhMakeFormControl(calculationTypes),
+      messageTypes: dhMakeFormControl(messageTypes),
+      gridAreas: dhMakeFormControl(gridAreas),
+      actorNumber: dhMakeFormControl(actorNumber),
+      status: dhMakeFormControl(status),
+      period: dhMakeFormControl(period),
+      created: dhMakeFormControl(created),
+      latestDispatch: dhMakeFormControl(latestDispatch),
     });
 
+    this.formReset$
+      .pipe(takeUntilDestroyed(this.destoryRef))
+      .subscribe(() => this.formGroup.reset(this.initial()));
```

Comment:
question: is there a need to reset the form this way? Doesn't the reset button do it properly?

Reply:
As fare as I can tell it only resets the search input field, and not the reset

Reply:
Hmmm ... just tested it without the `formReset$` and it seems to work. :)

Reply:
You are right, has been removed.